src/python/pants/backend/python/dependency_inference/rules.py

```diff
@@ -206,6 +206,33 @@ def _collect_imports_info(
     )
 
 
+def _filter_unowned_imports(
```

Comment:
I think the name should include why we're filtering, maybe `_remove_ignored_imports`?

Reply:
Indeed, a lot better, done. :)